python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -825,12 +823,13 @@ def get_two_hop_neighbors(self, start_vertices=None):
         _client = default_client()
 
         def _call_plc_two_hop_neighbors(sID, mg_graph_x, start_vertices):
-            return pylibcugraph_get_two_hop_neighbors(
+            results_ = pylibcugraph_get_two_hop_neighbors(
                 resource_handle=ResourceHandle(Comms.get_handle(sID).getHandle()),
                 graph=mg_graph_x,
                 start_vertices=start_vertices,
-                do_expensive_check=False,
+                do_expensive_check=True,
```

Comment:
Do we want to merge this with `do_expensive_check=True`?

Reply:
No. Let me fix that